lib/internal/web3_utils.dart

```diff
@@ -29,3 +40,51 @@ String getNetworkName(int chainId) {
       return 'Unknown Chain';
   }
 }
+
+enum LoginType {
+  metaMask,
+  google,
+}
+
+Uint8List bigIntToBytes(BigInt value, int length) {
```

Comment:
move eip 191 and eip 712 signature related to

web3_eip191.dart
web3_eip171.dart